From ee4621474fb78f33688567df1210f5610bd4d176 Mon Sep 17 00:00:00 2001
From: Zheng Junjie <zhengjunjie@iscas.ac.cn>
Date: Tue, 12 Nov 2024 16:27:31 +0800
Subject: [PATCH 2/4] adjust jazzy python version.

---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 693d9eda9..be3a51022 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -35,7 +35,7 @@ if($ENV{ROS_DISTRO} MATCHES "humble")
 elseif($ENV{ROS_DISTRO} MATCHES "iron")
   find_package(Python 3.10 EXACT REQUIRED COMPONENTS Development)
 elseif($ENV{ROS_DISTRO} MATCHES "jazzy")
-  find_package(Python 3.12 EXACT REQUIRED COMPONENTS Development)
+  find_package(Python 3.11 EXACT REQUIRED COMPONENTS Development)
 elseif($ENV{ROS_DISTRO} MATCHES "rolling")
   find_package(Python 3.12 EXACT REQUIRED COMPONENTS Development)
 endif()
-- 
2.48.1

