diff --git a/CMakeLists.txt b/CMakeLists.txt
index 693d9eda9..a9e62f146 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -31,7 +31,7 @@ find_package(TinyXML2 REQUIRED)
 find_package(yaml-cpp REQUIRED)
 
 if($ENV{ROS_DISTRO} MATCHES "humble")
-  find_package(Python 3.10 EXACT REQUIRED COMPONENTS Development)
+  find_package(Python 3.11 EXACT REQUIRED COMPONENTS Development)
 elseif($ENV{ROS_DISTRO} MATCHES "iron")
   find_package(Python 3.10 EXACT REQUIRED COMPONENTS Development)
 elseif($ENV{ROS_DISTRO} MATCHES "jazzy")
